src/customConfigTests/src/test/kotlin/org/vaccineimpact/orderlyweb/customConfigTests/GuestUserTests.kt

```diff
@@ -72,7 +73,7 @@ class GuestUserTests: SeleniumTest() {
 
         // user should see reports in report table
         rows = driver.findElements(By.cssSelector("table.dataTable tbody tr"))
-        assertThat(rows.count()).isEqualTo(2)
```

Comment:
the demo dir now contains a bunch of reports and we don't care how many rows there are exactly, just that reports are visible